cmd/gendeepcopy/deep_copy.go

```diff
@@ -104,7 +104,7 @@ func main() {
 	} else {
 		pkgname = gv.Group
 	}
-	if len(gv.Version) != 0 {
```

Comment:
> Why is it extraneous?

Retracted.

Waited so long for this: 'kruntime sounds like a hip hop label. Use "runtime"' - https://github.com/deads2k/origin/commit/c309a676734a80dfd7bc40361a16fbd851502ee9#commitcomment-9728700

Maybe I'll make an issue for it.  No need to touch it here, but I just had to say that.